### DISCUSSION TITLE
Raspberrypi cam / Record when motion to USB

### DISCUSSION THREAD

**Discussion by FIMARx** [Motion-Project/motion#1420] [Q&A] [answered]
Hi,

> So I took a fast look on this and installed motion. My camera is working fine but the resolution is really small. But that's not my issue. So I have a USB where I want to save footage when someone walk close to my room. I took some time searching about it but didn't find anything useful. Does anyone know any tutorial which can help me out? I just have a basic raspberrypi where I installed motion by the terminal. I used this tutorial https://www.instructables.com/Raspberry-Pi-remote-webcam/ to intsall the cam but didn't find any useful tutorial which explain how to save footage to USB only when there's something moving.

Thanks for the help and sorry for this. My first time ever trying this. 😇

- I don't have much knowledge on this stuff so sorry because I'm a noobie

**Comment by Mr-Dave** [comment]
The following are some videos on how to use the Motion software. [Link](https://www.youtube.com/channel/UCZpOjUMnc-uO3qIyZAD_4lg)

**Reply by FIMARx** [reply]
Hey,

Thanks a lot! I'll take an look for it. I'll answer here if I get it to work. 😊

**Reply by FIMARx** [reply]
Hey,

I've taken a look on the tutorials but still haven't figure it yet. My camera works normal. I made some configurations but it doesn't still work properly. I wanted to apply always when there's something moving around to capture it and to save it to my USB. I made the target dir `target_dir /media/pi/taltio` image width `1280` and height `800` as well `area_detect 4,5,6` `ffmpeg_output_movie on`  `ffmpeg_video_codec mpeg4` `locate_motion_mode on` but it doesn't save anything to my usb. It shows the usb name in terminal as `/media/pi/764C87F24C87AB85` so what should I put in the `target_dir` I guess I messed the save direction that's why it doesn't save anything to my usb. Thanks for helping 😊

**Reply by FIMARx** [reply]
Hey, 

I have figured it out. I used this tutorial https://raspberrypi.stackexchange.com/questions/12378/what-permissions-does-motion-require-to-write-to-specific-directory

It works to my local folder. But those commands didn't work on my usb drive, so I think it doesn't have the access to save the videos there. Is there any solution for this?

I mounted the drive and it's fat32 but still nothing.

**Comment by FIMARx** [comment, marked as answer]
I used this config and gave upped the usb thing. It didn't work for me tho. But it works fine without the usb save dir.

[motion.conf.txt](https://github.com/Motion-Project/motion/files/7408381/motion.conf.txt)
